projects/element-ng/entity-selection-dialog/si-entity-selection-dialog/si-entity-selection-dialog.component.scss

```
@@ -0,0 +1,26 @@
+// Temporary workaround to prevent all clients to update to
+// element-ng version 30.6.0 to make the multi click on row
+// selection view as desired. Can be removed later.
+::ng-deep {
+  .datatable-header-cell,
+  .datatable-body-cell {
```

even for a workaround, never do top-level `::ng-deep`. Instead (if still needed, version mentioned above is quite old) scope to host, i.e.


```suggestion
:host ::ng-deep {
  :is(.datatable-header-cell, .datatable-body-cell) {
```

and use `:is()` while at it for smaller CSS